### ISSUE TITLE
Using docker - readme update

### ISSUE THREAD

**Issue by marwin1991** [angrymeir/SecScanner2JUnit#26]
Update readme about using docker image? @angrymeir

**Comment by angrymeir** [comment]
Yeah it's weird...the repo is uploaded to dockerhub and accessible (everybody can pull it), but it's not indexed?
I have the same problem for another repository, so maybe its a dockerhub indexing issue?

Anyway: The repo is available here: https://hub.docker.com/r/angrymeir/secscanner2junit

**Comment by marwin1991** [comment]
I am wondering about `entrypoint', Can you explain me why do we need this?

![image](https://user-images.githubusercontent.com/25181517/195268411-6f56d2f9-fbb1-4b85-9d07-9f05f28b5de3.png)

**Comment by angrymeir** [comment]
In the dockerfile, I defined an entrypoint ([here](https://github.com/angrymeir/SecScanner2JUnit/blob/main/Dockerfile#L7)) so that one can also run the container manually like this:
`docker run -v /path/to/file:/tmp angrymeir/secscanner2junit:latest secrets infile.json outfile.xml`
instead of:
`docker run -it -v /path/to/file:/tmp angrymeir/secscanner2junit:latest ss2ju secrets infile.json outfile.xml`

Therefore GitLab CI has to override the existing entrypoint as otherwise it would fail. ([Entrypoint override documentation](https://docs.gitlab.com/ee/ci/docker/using_docker_images.html#override-the-entrypoint-of-an-image)).

**Comment by marwin1991** [comment]
Thank you for clarification 👍🏼

**Comment by angrymeir** [comment]
I have heard of other repositories, that are also not indexed correctly on dockerhub, so pretty sure it's an indexing issue on their side...
